package.json

```diff
@@ -1,5 +1,11 @@
 {
   "dependencies": {
-    "dotenv": "^16.0.0"
+    "@nomiclabs/hardhat-ethers": "^2.0.5",
+    "@nomiclabs/hardhat-etherscan": "^3.0.3",
+    "@nomiclabs/hardhat-waffle": "^2.0.3",
+    "@openzeppelin/contracts": "^4.5.0",
+    "dotenv": "^16.0.0",
+    "ethereum-waffle": "^3.4.4",
+    "hardhat": "^2.9.3"
```

Comment:
Do we need the hardhat dependencies in the main branch? 
You used them only for testing, right?

Reply:
you can leave `"dotenv": "^16.0.0"` in the file.

Reply:
Yes that is true we don't need hardhat dependencies, I will change that.